src/tablo/plugins/scrollable/scrollable.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import omit from 'lodash/omit';
+import t, { maybe } from 'tcomb';
+import { props } from 'tcomb-react';
+import { pure, skinnable, contains } from 'revenge';
+import cx from 'classnames';
+
+export default (Grid) => (
+  @props({
+    className: maybe(t.String),
+    scrollLeft: maybe(t.Integer),
+    scrollTop: maybe(t.Integer),
+    onScrollStart: maybe(t.Function),
+    onScrollEnd: maybe(t.Function)
+  }, { strict: false })
+  @pure
+  @skinnable(contains(Grid))
+  class ScrollableGrid extends React.Component {
+
+    state = {
+      scrollTop: this.props.scrollTop
+    }
+
+    componentWillReceiveProps(nextProps) {
+      const scrollTopDidntChanged = nextProps.scrollTop === this.props.scrollTop;
```

Comment:
to make functions more readable we usually pick every needed variable from `props`/`state` in the first line of the function:

```js
const { scrollTop, selectedRows } = this.props
```

(is it defined somewhere in our guidelines? not sure...)